tools/scripts/aducm.mk

```diff
@@ -223,9 +223,35 @@ $(MQTT_LIB):
 
 endif
 
+#	IIO
+IIO_DIR = $(NO-OS)/libraries/iio
+#If mqtt is found in SRC_DIRS
```

Comment:
mqtt ?
Perhaps a more general comment would be to make the comments in the makefile more homogenous in style and also make sure they are updated.

Reply:
copy paste miss :)

Reply:
I will come with a generic approach to link libraries in an other PR and I will put a generic comment there